app/views/org_admin/annotations/_form.html.erb

```diff
@@ -1,6 +1,7 @@
 <fieldset class="annotation_fields" id="<%= unique_dom_id(f.object, "fields") %>">
   <div class="form-group col-md-10">
-    <%= f.label(:type, f.object.type.humanize, class: "control-label") %>
```

Comment:
the text here is derived from the object type and not getting properly handled by get text